src/Sql/dbo/Stored Procedures/Organization_Update.sql

```diff
@@ -50,7 +50,8 @@
     @SmServiceAccounts INT = null,
     @MaxAutoscaleSmSeats INT = null,
     @MaxAutoscaleSmServiceAccounts INT = null,
-    @SecretsManagerBeta BIT = 0
+    @SecretsManagerBeta BIT = 0,
+    @LimitCollectionCdOwnerAdmin BIT = 1
```

Comment:
The `Update` SPROC having a different default than the `Create` one is probably even weirder than our previous discussion of having the `Create` SPROC differ from the default of the column. During the transition of the migration having run but the code having not been deployed, you could have an organization get created get the new value of false, but then have it's name (or anything else) get updated and it would flip it back to true.

Reply:
Again, I appreciate the perspective! After a conversation with @justindbaur and some analytical data, we're going to leave this as is for now due to our release tendencies and low exposure to the above scenario.